### DISCUSSION TITLE
Enabling Autologin

### DISCUSSION THREAD

**Discussion by sendmebits** [tteck/Proxmox#1425] [General]
I saw the pinned post about disabling the auto-login feature here - [#324](https://github.com/tteck/Proxmox/discussions/324)

However, I wanted to 'enable' auto-login, I could not find any posts or discussion here. I was able to find a way to enable it, with the below. Backup your LXC first, this could break things, generally I think you're supposed to avoid editing these files directly.

Edit this file: `/lib/systemd/system/container-getty\@.service`

Modify the ExecStart line to this:

`ExecStart=-/sbin/agetty --autologin root --noclear --keep-baud tty%I 115200,38400,9600 $TERM`

**Comment by tteck** [comment]
~Autologin will be executed through a LXC cmode setting in a forthcoming release, and once this change becomes effective, it cannot be undone.~

I have made the decision to continue making modifications to getty in order to achieve autologin.
